app/pages/cif/project-revision/[projectRevision]/index.tsx

```diff
@@ -115,6 +136,9 @@ export function ProjectRevision({
         div :global(.pg-button) {
           margin-right: 3em;
         }
+        h3 {
+          margin: 1em 0 0 0;
+        }
```

Comment:
If needed, this should be either moved to the individual form components, or changed to `div :global(h3)`. It won't do anything right now since this component doesn't render an h3 directly

Reply:
> a couple minor changes. Spacing will have to be revised when validation handling is added or when the themes are upgraded in #328

Should I add bullets about spacing to those tickets?

Reply:
sure, we want the label and value (`dt` and `dd`) to be on the same row, as pictured in the wireframes